lib/cadet_web/admin_controllers/admin_grading_controller.ex

```diff
@@ -58,6 +58,17 @@ defmodule CadetWeb.AdminGradingController do
     index(conn, %{"group" => "false"})
   end
 
+  def index_all_submissions(conn, _) do
+    index(
+      conn,
+      %{
+        "group" => "false",
+        "pageSize" => "100000000000",
```

Comment:
Haha sure, maybe in the future we can filter by assessment name (it's a use case for the PA because we export the grades to Canvas separately).

Reply:
oh yes, that would be a good future change to generalize the export function!